modules/user-guide/pages/collating-vale-output.adoc

```diff
@@ -0,0 +1,70 @@
+// Metadata for Antora
+:navtitle: Collating Vale output
+:keywords: collating, output
+:description: Collating the Vale CLI report results
+:page-aliases: end-user-guide:collating-vale-output.adoc
+// End of metadata for Antor
+
+:context: collating-vale-output
+:_module-type: CONCEPT
+[id="collating-vale-output_{context}"]
+= Collating Vale output
+
+As a technical writer, you can use a template to collate the results from running `vale` on the command line. This collated output is non-repetitive and might be easier to use.
```

Comment:
Is this "As a technical writer..." construction useful? Anyone can use the Vale at Red Hat style. 

```suggestion
You can use a template to collate the results from running `vale` on the command line. This collated output is non-repetitive and might be easier to use.
```

Reply:
Updated. Thanks.